test/cpp/jit/tests_setup.py

```diff
@@ -51,9 +51,22 @@ def setup(self):
         torch.save(value, self.path, _use_new_zipfile_serialization=True)
 
 
+class TorchSaveError(FileSetup):
```

Comment:
can you put a comment token in so it is possible to find the other half of this test from tests.h? `// [testTorchSaveError]`